examples/pipelines/hf_dataset_pipeline/README.md

```diff
@@ -0,0 +1,63 @@
+In this guide, we explain how to implement a KubeFlow pipeline and run it on KubeFlow on GKE (Google Kubernetes Engine).
```

Comment:
The name 'hf_dataset_pipeline` is a bit abstract and can indicate many things. Can we change it to Stable diffusion data collection or finetuning?

Reply:
Depends on what we want to showcase. If this example is meant to showcase how to work with a HuggingFace dataset, I think the current name is appropriate.

Reply:
Going to keep the name for now